### PR TITLE
IME-63 To allow Setting up two network devices. These devices can be …

### PR DESCRIPTION
**Pull request by manfrednde**
…static or DHCP

### CODE REVIEW

**Comment by rackhdmirror** [comment]
@RackHD/corecommitters please review.

**Comment by manfrednde** [comment]
The changes will allow user to set up network devices with static or dynamic ip address. it will also allow to create switch devices.
@RackHD/corecommitters @jimturnquist @johren

File: data/templates/esx-ks [hunk 1]
```diff
@@ -171,6 +183,9 @@ cp /var/log/esxi_install.log "/vmfs/volumes/datastore1/firstboot-esxi_install.lo
       esxcli network ip interface remove -i <%=vmkname%>
       esxcli network ip interface add -i <%=vmkname%> -p <%=n.device%>
       esxcli network ip interface ipv4 set -i <%=vmkname%> -t dhcp
+      sleep 60
```
Comment (yyscamper) [inline data/templates/esx-ks]: Could you please explain why there is a delay here?

Comment (manfrednde) [inline data/templates/esx-ks]: To allow the interface to dhcp and gets an IP address before we can query that IP to post lookup.

**Pull request merged**